docs/installation/integrations/next-js.md

```diff
@@ -0,0 +1,204 @@
+---
+menu-title: Next.js
+meta-title: Compatibility with Next.js | CKEditor 5 documentation
+meta-description: Learn how to integrate the Next.js framework with the rich text editor - CKEditor 5 - using both the App Router and Pages Router routing strategies.
+category: integrations
+order: 100
+modified_at: 2023-11-14
+---
+
+# Compatibility with Next.js
+
+Next.js is a React meta-framework that helps create full-stack web apps for production. It offers different rendering strategies like server-side rendering (SSR), client-side rendering (CSR), or static site generation (SSG). Additionally, it provides file-based routing, automatic code splitting, and other handy features out of the box.
```

Comment:
~for production~ not needed IMHO.

Reply:
Next.js was marketing itself as a "React framework for production" - that was a pretty good summary, IMO. But we can omit it.